MAINTAINERS

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Please put this in a separate commit, so that the logical change units "fix tests", and "take over maintainership" are clearly delineated, and documented individually in their respective commit messages.